src/Processors/Transforms/buildPushingToViewsChain.cpp

```diff
@@ -493,7 +506,8 @@ static QueryPipeline process(Block block, ViewRuntimeData & view, const ViewsDat
         pipeline = interpreter.buildQueryPipeline();
     }
     else
-    {   InterpreterSelectQuery interpreter(view.query, local_context, SelectQueryOptions());
+    {
+        InterpreterSelectQuery interpreter(view.query, local_context, SelectQueryOptions());
```

Comment:
`InterpreterSelectQuery` checks that view source matches the query inside. But I suppose it's better to have it on `buildPushingToViewsChain` level.